joomla/joomla-cms
# [5.3] Upmerge changes from 5.2-dev 2024-11-11

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) merges the changes from 5.2-dev up into 5.3-dev which have been made since the last upmerge with PR #44367 , which includes the file permissions (mode) changes in the git index of the repository from commit https://github.com/joomla/joomla-cms/commit/9b13ba25eeedb04f10facab91e385ca9e13b4c94 .

In addition this PR fixes the javascript-cs step failing in Drone due to a missing trailing comma in file tests/System/plugins/fs.mjs . In the 5.2-dev branch the linter doesn't complain about that, but here it does. I have created PR #44435 for the 5.2-dev branch to fix it there, too, and will later handle the conflict when doing the next upmerge for 5.3-dev.

### Testing Instructions

Review. Check the commits of this PR and those in the 5.2-dev branch, and check the code changes and the file permissions (mode) changes in this PR.

### Link to documentations
Please select:
